js/bootstrap-colorpicker-module.js

```diff
@@ -113,10 +113,10 @@ angular.module('colorpicker.module', [])
           V = Math.max(r, g, b);
           C = V - Math.min(r, g, b);
           H = (C === 0 ? null :
-              V === r ? (g - b) / C :
-                  V === g ? (b - r) / C + 2 :
-                      (r - g) / C + 4
-              );
+            V === r ? (g - b) / C :
+            V === g ? (b - r) / C + 2 :
```

Comment:
This is a super weird function, but for readability we still want those indents:

```
H = (C === 0 ? null :
  V === r ? (g - b) / C :
    V === g ? (b - r) / C + 2 :
      (r - g) / C + 4
);
```